portfolio/src/main/webapp/script.js

```diff
@@ -64,3 +64,239 @@ function clearComments() {
+          },
+          {
+            featureType: 'poi.park',
+            elementType: 'labels.text.fill',
+            stylers: [{color: '#6b9a76'}]
+          },
+          {
+            featureType: 'road',
+            elementType: 'geometry',
+            stylers: [{color: '#38414e'}]
+          },
+          {
+            featureType: 'road',
+            elementType: 'geometry.stroke',
+            stylers: [{color: '#212a37'}]
+          },
+          {
+            featureType: 'road',
+            elementType: 'labels.text.fill',
+            stylers: [{color: '#9ca5b3'}]
+          },
+          {
+            featureType: 'road.highway',
+            elementType: 'geometry',
+            stylers: [{color: '#616163'}]
+          },
+          {
+            featureType: 'road.highway',
+            elementType: 'geometry.stroke',
+            stylers: [{color: '#1f2835'}]
+          },
+          {
+            featureType: 'road.highway',
+            elementType: 'labels.text.fill',
+            stylers: [{color: '#f3d19c'}]
+          },
+          {
+            featureType: 'transit',
+            elementType: 'geometry',
+            stylers: [{color: '#2f3948'}]
+          },
+          {
+            featureType: 'transit.station',
+            elementType: 'labels.text.fill',
+            stylers: [{color: '#d59563'}]
+          },
+          {
+            featureType: 'water',
+            elementType: 'geometry',
+            stylers: [{color: '#17263c'}]
+          },
+          {
+            featureType: 'water',
+            elementType: 'labels.text.fill',
+            stylers: [{color: '#515c6d'}]
+          },
+          {
+            featureType: 'water',
+            elementType: 'labels.text.stroke',
+            stylers: [{color: '#17263c'}]
+          }
+        ]
+      });
+
+  // Variables starting with shabu will be referring to "Hose of Shabu Shabu II"
+  const shabuContentString = '<div id="shabu-content">'+
+    '<h1 class="firstHeading">House of Shabu Shabu II</h1>'+
+    '<div>'+
+    '<p><b>House of <a href="https://en.wikipedia.org/wiki/Shabu-shabu">Shabu Shabu</a> II</b> is a great place to '+
+    'go to with a friend or a special someone, especially if you are into '+
+    'the interactive experience of <a href="https://en.wikipedia.org/wiki/Korean_barbecue">Korean barbecue</a>. '+
+    'The experience at House of Shabu Shabu II is similar in that '+
+    'you choose from a selection of menus which include a multitude '+
+    'of meats and vegetables and can order as much as you want! </p> '+
+    '</div>'+
+    '</div>';
+
+  // Variables starting with boiling will be referring to "Boiling Point"
+  const boilingContentString = '<div id="boiling-content">'+
+    '<h1 class="firstHeading">Boiling Point</h1>'+
+    '<div>'+
+    '<p><b>Boiling Point</b> is an establishment that specializes in <a href="https://en.wikipedia.org/wiki/Hot_pot">Taiwanese hot soup</a> cuisine. '+
```

Comment:
Same here and below when it comes to storing URL strings in variables.

Reply:
Done.